subprojects/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/dynamic_versions.adoc

```diff
@@ -24,10 +24,7 @@ Alternatively, the module you request can change over time even for the same ver
 An example of this type of _changing module_ is a Maven `SNAPSHOT` module, which always points at the latest artifact published.
 In other words, a standard Maven snapshot is a module that is continually evolving, it is a "changing module".
 
-[NOTE]
-====
-Using dynamic versions and changing modules can lead to unreproducible builds. As new versions of a particular module are published, its API may become incompatible with your source code. Use this feature with caution!
-====
+NOTE: Using dynamic versions and changing modules can lead to unreproducible builds. As new versions of a particular module are published, its API may become incompatible with your source code. Use this feature with caution!
```

Comment:
🤔 As this note literally ends with caution, perhaps we should switch it to a `CAUTION` admonition?